L64-python-runtime-proto-parsing.md

```diff
@@ -0,0 +1,99 @@
+gRPC Python Runtime Proto Parsing
+----
+* Author(s): gnossen
+* Approver: lidizheng
+* Status: Draft
+* Implemented in: Python
+* Last updated: April 9, 2020
+* Discussion at: **TODO(gnossen)**
```

Comment:
We might still have the 10 days open discussion restriction before merging the PR?

Reply:
Yes. I wanted to some eyeballs on it as a sanity check before creating the thread. Added!